src/common/base/tests/it/runtime.rs

```diff
@@ -77,3 +81,43 @@ async fn test_shutdown_long_run_runtime() -> Result<()> {
 
     Ok(())
 }
+
+lazy_static! {
```

Comment:
Perhaps we can rewrite it using once_cell.

So that in the future we can migrate painlessly from once_cell to std